### PR TITLE
trading modes migration adaptations 

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot-Backtesting/pull/52 in octobot-backtesting version
requires https://github.com/Drakkar-Software/OctoBot-Backtesting/pull/54

### CODE REVIEW

File: octobot_trading/api/symbol_data.py [hunk 1]
```diff
@@ -44,3 +44,7 @@ def create_new_candles_manager(candles=None) -> CandlesManager:
     if candles is not None:
         manager.replace_all_candles(candles)
     return manager
+
+
+def force_set_mark_price(exchange_manager, symbol, price):
```
Comment (Herklos) [inline octobot_trading/api/symbol_data.py:49]: What will be the usage of this method ?

Comment (GuillaumeDSM) [reply octobot_trading/api/symbol_data.py:49]: Setting the price for testing ordre creation (trading mode tests)

Comment (Herklos) [reply octobot_trading/api/symbol_data.py:49]: OK if it's only for testing purposes :+1:

File: octobot_trading/producers/abstract_mode_producer.py [hunk 2]
```diff
@@ -130,40 +128,6 @@ def get_should_cancel_loaded_orders(cls):
         """
         raise NotImplementedError("get_should_cancel_loaded_orders not implemented")
 
-    async def create_order_if_possible(self) -> None:
```
Comment (Herklos) [inline octobot_trading/producers/abstract_mode_producer.py:133]: :+1:

File: tests/modes/test_abstract_mode_consumer.py [hunk 3]
```diff
@@ -0,0 +1,166 @@
+#  Drakkar-Software OctoBot-Trading
```
Comment (Herklos) [inline tests/modes/test_abstract_mode_consumer.py:1]: Nice !

**Pull request merged**